src/main/java/org/terracotta/statistics/jsr166e/LongAdder.java

```diff
@@ -33,7 +33,7 @@
  * collection keys.
  *
  * <p><em>jsr166e note: This class is targeted to be placed in
- * java.util.concurrent.atomic<em>
+ * java.util.concurrent.atomic</em>
```

Comment:
Not too fond of modifying these classes which have been ported from another code base. Especially to please the Java 8 javadoc.